Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_slippy_tiles"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
[MEMO] I bumped the version to `0.10.1` to align with the `deprecated` macros, though I understand this is typically done during release.

https://github.com/hiratara/bevy_slippy_tiles/blob/f99e3c38f1e7587e33104ebcd32314a92d604e5f/src/download.rs#L168